### DISCUSSION TITLE
Could not open github codespaces, wrong with some `nls` issues.

### DISCUSSION THREAD

**Discussion by liudonghua123** [community/community#137455] [Codespaces]
### Select Topic Area

General

### Body

Hi, I couldn't open codespaces recently, I checked the Console, Network of Devtools. And I found it's related to some `nls` requests ([nls.messages.js](https://www.vscode-unpkg.net/nls/fee1edb8d6d72a0ddff41e5f71a671c23ed924b9/1.92.2/en/nls.messages.js), [workbench.web.main.nls.js](https://assets.github.dev/stable/fee1edb8d6d72a0ddff41e5f71a671c23ed924b9/out/vs/workbench/workbench.web.main.nls.js)) could not get.

![image](https://github.com/user-attachments/assets/a7d699d8-e3f0-4c22-a7f9-4d833808ef76)

![image](https://github.com/user-attachments/assets/00f40a01-1618-48ad-8491-8aa6d16f5976)

![image](https://github.com/user-attachments/assets/66150e4e-e1b5-4419-afe2-b8eb2e051e55)

**Comment by zaclummys** [comment]
Same problem here:
![image](https://github.com/user-attachments/assets/a98d3517-fcf0-4cdb-9a46-db42af30be5d)
